src/Test.tsx

```diff
@@ -5,6 +5,7 @@
 import React from 'react';
```

Comment:
Seems that the entire file is now not used really, is it?

Reply:
right, this need to be removed.